config/services/storage/service.json

```diff
@@ -4,8 +4,5 @@
 		"ram": 256,
```

Comment:
I wonder if *this* could use a bump. See this (NOVERs for a storage zone) in nightly-2, for example:

```
[root@headnode (nightly-2) ~]# zonememstat -a
                                 ZONE                                         ALIAS  RSS(MB) CAP(MB)    NOVER  POUT(MB) SWAP%
...
 0a1dc7ec-5c44-420e-a22b-a4c7dc5946b2            storage.nightly.joyent.us-0a1dc7ec      236    256      151        26 44.3413
 9707c6ef-1c8a-4f6b-8861-eba8bc3cb321           storinfo.nightly.joyent.us-9707c6ef      178    256        0         0 29.7279
```

Reply:
In perf lab, the rss is somewhere between 250-300. I've bumped it up to 512 which should be suffix for most dev/test labs.